projects/aca-content/folder-rules/src/manage-rules/manage-rules.smart-component.spec.ts

```diff
@@ -143,9 +168,9 @@ describe('ManageRulesSmartComponent', () => {
 
     expect(component).toBeTruthy();
 
-    const adfEmptyContent = debugElement.query(By.css('adf-empty-content'));
-    const ruleSets = debugElement.queryAll(By.css(`[data-automation-id="rule-set-list-item"]`));
-    const ruleDetails = debugElement.query(By.css('aca-rule-details'));
+    const adfEmptyContent = unitTestingUtils.getByCSS('adf-empty-content');
```

Comment:
adf-empty-content is selector of component so please use getByDirective function instead to get that element